### ISSUE TITLE
Support newer version of Python

### ISSUE THREAD

**Issue by melouaer** [aws-cloudformation/cloudformation-cli-python-plugin#263]
Hello team

Cloudformation cli python plugin supports python3.6 and python3.7.
As of today, these python versions are at end of support.
Are there any plans to support a newer version of Python runtime in the roadmap? 

Thank you

**Comment by kapilt** [comment]
yeah this causes a serious issue wrt to using dependencies as most of the ecosystem has moved on from these unsupported versions.

**Comment by kapilt** [comment]
it does look like newer versions are supported per the cli code just the documentation is woefully out of date.

https://github.com/aws-cloudformation/cloudformation-cli/blob/master/src/rpdk/core/hook/init_hook.py
